### PR TITLE
Add client flags to turn dnd on/off

### PR DESCRIPTION
**Pull request by bvergnaud**
Hello 👋🏻 

This is a QoL PR I offer, for which I'll be the first client. Unless I missed something, setting dnd state to on or off currently requires getting the current dnd state, and toggling if necessary. With this MR, it'll be doable in a single call to the client. Less bash scripts everywhere can only be a good thing for the world. 😛   

Fair warning:
* This is my first time poking in this codebase. I might have missed things. Feel free to tell me if anything needs tweaking.
* Up until today, I'd never even heard of vala. The edit is small and simple, and the code seems pretty straightforward, but reviewers should assume it's been written by a newbie, I won't take it personally. 😄

Thanks !

### CODE REVIEW

File: completions/zsh/_swaync-client [hunk 1]
```diff
@@ -10,6 +10,8 @@ _arguments -s \
     '(-cp --close-panel)'{-cp,--close-panel}'[Closes the notificaion panel]' \
     '(-d --toggle-dnd)'{-d,--toggle-dnd}'[Toggle and print the current dnd state]' \
     '(-D --get-dnd)'{-D,--get-dnd}'[Print the current dnd state]' \
+    '(-dn --dnd-on)'{-dn --dnd-on}'[Turn dnd on and print the new dnd state]' \
+    '(-df --dnd-off)'{-df --dnd-off}'[Turn dnd off and print the new dnd state]' \
```
[the remark this answers is not in the document]
Comment (bvergnaud) [reply completions/zsh/_swaync-client]: Indeed, good catch. I pushed an update.

File: src/client.vala [hunk 2]
```diff
@@ -161,6 +163,18 @@ public int command_line (string[] args) {
             case "-D":
                 print (cc_daemon.get_dnd ().to_string ());
                 break;
+            case "--dnd-on":
+            case "-dn":
+                if (cc_daemon.get_dnd () == false) {
+                    print (cc_daemon.toggle_dnd ().to_string ());
+                }
```
[the remark this answers is not in the document]
Comment (bvergnaud) [reply src/client.vala]: That's a fair improvement. I missed the existence of that method and assumed I'd have to work with the toggle method. But your suggestion is better. I pushed an update.

[the remark this answers is not in the document]
Comment (ErikReider) [reply src/client.vala]: All of the available methods are here: [SwayncDaemon](https://github.com/ErikReider/SwayNotificationCenter/blob/b65a08bb27bc584b11b15788a09884c1f2e668ba/src/swayncDaemon/swayncDaemon.vala)

**Review by ErikReider — APPROVED** [review]
LGTM! Thanks for the help! :)

**Pull request merged**